src/ensembl/src/shared/components/communication-framework/contact-us/ContactUs.tsx

```diff
@@ -29,13 +36,63 @@ const ContactUs = () => {
     @Imran: update the below accordingly
   */
   return (
-    <div>
-      <p>
-        Please contact us if you have a problem with the website or need help
-      </p>
-      <SecondaryButton onClick={() => setShouldShowForm(!shouldShowForm)}>
-        Contact us
-      </SecondaryButton>
+    <div className={styles.wrapper}>
+      <div>
+        <p>
+          Please contact us if you have a problem with the website or need help
+        </p>
+        <SecondaryButton onClick={() => setShouldShowForm(!shouldShowForm)}>
+          Contact us
+        </SecondaryButton>
+      </div>
+      <div>
+        <section className={styles.socialMediaInfo}>
+          <p>
+            Visit the blog for details of releases, workshops and other tidbits
+            of information about the Ensembl project.
+          </p>
+          <p>
+            Visit Facebook or Twitter for the latest breaking Ensembl news and
+            service status updates.
+          </p>
+          <p>
+            Ensembl Blog <BlogIcon className={styles.icon} />{' '}
+            <FacebookIcon className={styles.icon} />{' '}
+            <TwitterIcon className={styles.icon} />
+          </p>
+        </section>
+        <section>
+          <h4>Online Help</h4>
```

Comment:
I'm in general very confused about these heading tags. Couldn't find any guidance on the web for how to think about headings, but... why are they heading 4's?

On the one hand, the purpose of the headings is semantics and we shouldn't be influenced by how they are automatically styled by the browser, because we can easily style them for our specific needs.

On the other hand, I find it hard to use an h1 here.

Still... Why an h4? 😄

Reply:
Hmm that's true. I thought these headings don't have a high importance in the whole page. Probably `h2` would be preferable here?

Reply:
I've changed these heading tags to `h2`